zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt

```diff
@@ -188,6 +189,13 @@ class DocumentenApiPlugin(
         informationObjectType: String,
         storedDocumentUrl: String
     ) {
+        //determine size of InputStream if not null
+        val contentInBytes = contentAsInputStream.readAllBytes()
+        IOUtils.closeQuietly(contentAsInputStream)
+        val bestandsOmvang = contentInBytes.size.toLong()
+        val copyInputStream = contentInBytes.inputStream()
```

Comment:
This is tricky. We made this based on streams to prevent having to load the entire file into memory when it comes to big files. This way we still do that anyway. Do we not have some meta data on what the file size is? Can we get that from the temporary file storage for example?

Reply:
@ivo-ritense I have refactored the solution with using metadata for the filesize